binding.js

```diff
@@ -1 +1 @@
-module.exports = require('node-gyp-build')(__dirname)
+module.exports = require('require-addon')('.', __filename)
```

Comment:
```suggestion
require.addon = require('require-addon')

module.exports = require.addon('.', __filename)
```

For context: https://github.com/holepunchto/require-addon/commit/e990dfbe90df3c7fe2aa654cecb957dd9d6ed670